java/ql/src/experimental/Security/CWE/CWE-020/Log4jJndiInjection.qhelp

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Log4j versions prior to 2.15.0 are subject to a remote code execution vulnerability via the ldap JNDI parser.
+</p>
+<p>
+As per Apache's Log4j security guide: Apache Log4j2 &lt;=2.14.1 JNDI features used in configuration, log messages, and parameters 
+do not protect against attacker controlled LDAP and other JNDI related endpoints. An attacker who can control log messages or 
+log message parameters can execute arbitrary code loaded from LDAP servers when message lookup substitution is enabled. 
+From log4j 2.15.0, this behavior has been disabled by default.
+</p>
+</overview>
+
+<recommendation>
+<p>
+This issue was remediated in Log4J v2.15.0. The Apache Logging Services team provides the following mitigation advice:
```

Comment:
```suggestion
This issue was remediated in Log4j v2.15.0. The Apache Logging Services team provides the following mitigation advice:
```

Reply:
Let's follow Apache's [own guidelines](https://logging.apache.org/log4j/2.x/) on formatting "Log4j": uppercase "L" and lowercase "j".